styles.css

```diff
@@ -112,40 +112,268 @@ section {
   gap: 20px;
 }
 
+.work {
+  padding: 114px 10%;
+  align-items: center;
+}
+
+.container_work {
+  display: grid;
+  grid-template-columns: 1fr;
+  align-items: center;
+  row-gap: 88px;
+}
+
+.work_content {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  gap: 12px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+.work_description {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0;
+  gap: 12px;
+}
+
+.work_description h2 {
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 44px;
+  color: #172b4d;
+}
+
+.conduct {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 12px;
+}
+
+.conduct_head {
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+}
+
+.conduct_para {
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: black;
+}
+
+.conduct_year {
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: black;
+}
 
-@media screen and (min-width: 768px) {
-  .contact img {
-    display: none;
-  }
-
-  .body {
-    height: 100vh;
-  }
-
-  .desktop-header {
-    display: flex;
-    flex-direction: row;
-    list-style-type: none;
-    gap: 32px;
-    justify-content: space-between;
-    padding: 10px 0 10px 10px;
-    animation: fadeIn 3s ease-in;
-    animation-delay: 2s;
-    animation-iteration-count: 2;
-  }
-
-  .navigation .nav {
-    display: none;
-  }
-
-  .headline {
-    height: 100vh;
-    background-image: url("./image/Header\ bg@2x.svg");
-    background-repeat: no-repeat;
-    background-position-x: 100%;
-    background-position: 100%;
-    background-position: 100% 100%;
-  }
-
- 
+.work_paragraph {
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.work_languages {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 8px;
+}
+
+.work_languages li {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  padding: 4px 12px;
+  border-radius: 8px;
+  background: #ebebff;
+}
+
+.item > img {
+  width: 10px;
+}
+
+.contact img {
+  position: absolute;
+  height: 709.17px;
+  height: 100%;
+  right: 0;
+  bottom: 0;
+}
+
+.btn {
+  font-weight: 500px;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  padding: 10px;
+  background: #fff;
+  border: 2px solid #87a0de;
+  border-radius: 8px;
+  transition: color;
+  transition-duration: 1s;
+  transition-delay: 0.1s;
+}
```

Comment:
- [ ] As requested by the previous reviewer please make sure you've implemented the `:hover` and `:active` (pressed) state interaction of this button as specified in the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23-9&t=t45uZqimwMP5BSMR-0) 🎨. 
    - Here's the Figma design as a reminder 😉:
      ![firefox_7KIb3eS8J1](https://user-images.githubusercontent.com/104968236/229153208-05a20af2-e6dd-4037-a845-a828efe72e8b.png)
- You can refer to this [article here](https://bitsofco.de/when-do-the-hover-focus-and-active-pseudo-classes-apply/) if you need more info.